activerecord/lib/active_record/relation.rb

```diff
@@ -641,6 +641,13 @@ def inspect
       "#<#{self.class.name} [#{entries.join(', ')}]>"
     end
 
+    protected
+
+      def load_records(records)
```

Comment:
@sgrif Please have a look at this protected method regarding loading records into the relation.